medusa/search/manual.py

```diff
@@ -300,8 +300,8 @@ def get_provider_cache_results(series_obj, show_all_results=None, perform_search
             i['pretty_size'] = pretty_file_size(i['size']) if i['size'] > -1 else 'N/A'
             i['seeders'] = i['seeders'] if i['seeders'] >= 0 else '-'
             i['leechers'] = i['leechers'] if i['leechers'] >= 0 else '-'
-            i['pubdate'] = sbdatetime.convert_to_setting(parser.parse(i['pubdate'])).strftime(
-                app.DATE_PRESET + ' ' + app.TIME_PRESET) if i['pubdate'] else '-'
+            i['pubdate'] = parser.parse(i['pubdate']).astimezone(app_timezone).strftime(
+                '%Y-%m-%d %H:%M:%S') if i['pubdate'] else '-'
```

Comment:
The code looks good to me, but why did you remove the time and date preset? It was meant to show the pubdate as set in the config.

Reply:
![image](https://user-images.githubusercontent.com/10238474/37965428-c57d6e6a-31cd-11e8-8d56-a2c10d6ed705.png)

None of the date formats matched.
If I change one, I'll want to change the others... But what is ideal? Should I apply the user's preference for "timeago" to all datetimes here?

Reply:
I still think the issue is in the table sort parsing, in the javascript fle

Reply:
Also, Published is the only column that is unsortable (of the datetime columns), probably because we need to have a timestamp for each one in the code - so that tableSorter can easily sort them.

Reply:
@OmgImAlexis which file does the tablesorting parsing for date fields in the manual search?

Reply:
https://github.com/pymedusa/Medusa/blob/be284322b5b0ed8cf8624a3827f3210501573335/themes-default/slim/static/js/home/snatch-selection.js#L75-L97

Reply:
I can't find this code for manual search table sorter:

https://github.com/pymedusa/Medusa/blob/master/themes/light/assets/js/home/index.js#L103-L126

Reply:
I'm working on it again now. I also noticed if you try to sort by quality it's not sorting correctly. Fixing tableSorter config should fix this as well.